modules/xray/shared/src/main/scala/natchez/xray/XRaySpan.scala

```diff
@@ -53,28 +54,28 @@ private[xray] final case class XRaySpan[F[_]: Sync](
 
   def spanId: F[Option[String]] = segmentId.some.pure[F]
 
-  def traceUri: F[Option[URI]] = Option.empty.pure[F]
+  def traceUri: F[Option[URI]] = none[URI].pure[F]
 
-  private def toEpoch(t: Instant): Double =
-    t.getEpochSecond().toDouble + t.getNano().toDouble / 1000000000
+  private def toEpoch(t: FiniteDuration): Double =
+    t.toSeconds.toDouble + t.toNanos.toDouble / 1000000000
```

Comment:
Glad we could change this, but ... are you sure these do the same thing?

The API for `Instant.getNano()` says:
> Gets the number of nanoseconds, later along the time-line, from the start of the second. The nanosecond-of-second value measures the total number of nanoseconds from the second returned by getEpochSecond.

But `FiniteDuration.toNanos` says:
> Return the length of this duration measured in whole nanoseconds, rounding towards zero.

Reply:
I think you probably just want `t.toNanos.toDouble / 1000000000`.

Reply:
Or maybe just `t.toUnit(TimeUnit.SECONDS)`

Reply:
The [X-Ray documentation](https://docs.aws.amazon.com/xray/latest/devguide/xray-api-segmentdocuments.html#api-segmentdocuments-fields) suggested using microseconds, so I went with that. Good catch!